py_proxy/views.py

```diff
@@ -0,0 +1,23 @@
+"""The views for the Pyramid app."""
+from pyramid import response, view
+
+
+@view.view_config(renderer="py_proxy:templates/index.html.jinja2", route_name="index")
+def index(request):
+    """Index endpoint."""
+    # pylint: disable=unused-argument
```

Comment:
You can also rename the argument to `_request`, then you don't need the pylint comment